### PR TITLE
Creating Azure Identity governance ELM team's logic app sample to be able to use in ELM feature called extensibility.

### PR DESCRIPTION
**Pull request by vikama**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*This is a new folder to help Azure Identity , Entitlement management teams new feature called extensibility. 
*We only support logic app as customers endpoints, so this template help customers to create a logic app with all the necessary information that is need to be able to use in ELM.

### CODE REVIEW

**Comment by vikama** [comment]
Adding Azure Identity , ELM team extensibility custom action logic app sample

**Comment by vikama** [comment]
closed bymistake

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Deploy a logic app that can be used in Azure Identity Governance ELM CustomAction
+
+ [![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fvikama%2FELMCustomAction%2Fmain%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fvikama%2FELMCustomAction%2Fmain%2Fazuredeploy.json)
+
+This template create a simple logic app with all the auth settings and schema of the http trigger that is needed my ELM Custom action API.
```
Comment (bmoore-msft) [inline 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/README.md:6]: fix typo "by ELM custom action ..."

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      },
+      "location":{
+         "type":"String",
+         "defaultValue":"westcentralus",
+         "metadata":{
+            "Description":"Deployment location"
+         }
+      }
+   },
+   "variables":{
```
Comment (bmoore-msft) [inline 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json:20]: remove variables property

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+            "accessControl":{
+               "triggers":{
+                  "openAuthenticationPolicies":{
+                     "policies":{
+                        "ELMAuthPolicy":{
+                           "type":"AAD",
+                           "claims":[
+                              {
+                                 "name":"iss",
+                                 "value":"https://sts.windows.net/72f988bf-86f1-41af-91ab-2d7cd011db47/"
```
Comment (bmoore-msft) [inline 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json:147]: will this work in all tenants?

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+                              },
+                              {
+                                 "name":"aud",
+                                 "value":"https://management.core.windows.net/"
```
Comment (bmoore-msft) [inline 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json:151]: use the environment function to get the endpoint

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+                              },
+                              {
+                                 "name":"appid",
+                                 "value":"810dcf14-1858-4bf2-8134-4c369fa3235b"
```
Comment (bmoore-msft) [inline 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json:155]: what app is this?  (add details to the readme)

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.parameters.json [hunk 6]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (bmoore-msft) [inline 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.parameters.json:1]: add $schema property, remove params that have defaultValues

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/metadata.json [hunk 7]
```diff
@@ -0,0 +1,10 @@
+{
+  "itemDisplayName": "Deploy a sample logic app that can be used in Azure Identity Governance ELM CustomAction",
+  "description": "This template create a simple logic app with all the auth settings and schema of the http trigger that is needed my ELM Custom action API.",
```
Comment (bmoore-msft) [inline 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/metadata.json:3]: fix typos

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json (hunk 3 above)
Comment (vikama) [reply 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json]: updating this

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json (hunk 5 above)
Comment (vikama) [reply 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.json:152]: This is azure ELM first party app Id will explain in Read me about this

**Comment by vikama** [comment]
Hi Brian,

Resolved all comments and fixed the build errors also , everything looks good, can you please take a look when you get a change ?

Thanks,

Bhaskar

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.parameters.json [hunk 8]
```diff
@@ -1,8 +1,12 @@
...
-      "value":"westcentralus"
-   }
+	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"name": {
+			"value": "ELM-custom-action"
+		},
+		"catalogId": {
+			"value": ""
```
Comment (bmoore-msft) [inline 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.parameters.json:9]: if this is not required - remove the param from here and add a defaultValue of emptyString in the template - also add a note to the description saying it's optional.

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/README.md [hunk 9]
```diff
@@ -1,13 +1,21 @@
...
 
-| Name   | Description    |
-|:--- |:---|
-| name  | Name of the Logicapp. |
-| location  | Location for the Logicapp.  |
+ [![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp%2Fazuredeploy.json)
+
+This template create a simple logic app with all the authorization policy settings and schema to http trigger that is needed by ELM custom action API.
```
Comment (bmoore-msft) [inline 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/README.md:15]: "This template creates..."

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/metadata.json [hunk 10]
```diff
@@ -1,10 +1,12 @@
 {
-  "itemDisplayName": "Deploy a sample logic app that can be used in Azure Identity Governance ELM CustomAction",
-  "description": "This template create a simple logic app with all the auth settings and schema of the http trigger that is needed my ELM Custom action API.",
-  "summary": "Deploy a sample logic app that can be used in Azure Identity Governance ELM CustomAction",
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Deploy a sample logic app, to use as ELM custom action",
+  "description": "This template create a simple logic app with all the authorization policy settings and schema to http trigger that is needed by ELM custom action API.",
```
Comment (bmoore-msft) [inline 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/metadata.json:5]: This template creates...

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.parameters.json (hunk 8 above)
Comment (vikama) [reply 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.parameters.json]: This is required and so left it here but updated with a sample Id which is a Guid

Comment (bmoore-msft) [reply 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.parameters.json]: What does the user need to provide?  A specific GUID?  If so, state what that is in the readme (or how the user gets the GUID).  Or any GUID?  If so, should it be idempotent?  If so, use the guid() function in the defaultValue of the param, if not use the newGuid() function.

Comment (vikama) [reply 101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.parameters.json]: this is not any guid from guid from ELM , so updated the readme to have this explained.

**Pull request merged**